### PR TITLE
Add objDesc tag

### PR DESCRIPTION
**Pull request by antirek**
add encoding/decoding Object Descriptor tag

0x07 tag by X.690 asn.1 types

### CODE REVIEW

File: lib/asn1/base/node.js [hunk 1]
```diff
@@ -170,7 +170,6 @@ tags.forEach(function(tag) {
 });
 
 Node.prototype.use = function use(item) {
-  assert(item);
```
Comment (indutny) [inline lib/asn1/base/node.js]: What is the motivation behind this?

**Comment by antirek** [comment]
Please, sorry. It's my mistake.

**Comment by indutny** [comment]
Landed and released in 4.9.0!

**Comment by indutny** [comment]
Thank you 😉

**Pull request closed without merging**